libmamba/src/solver/libsolv/database.cpp

```diff
@@ -39,7 +39,7 @@ namespace mamba::solver::libsolv
         Matcher matcher;
     };
 
-    Database::Database(specs::ChannelResolveParams channel_params)
+    Database::Database(specs::ChannelResolveParams channel_params, bool add_logger)
```

Comment:
Note to reviewers: I do not think `add_loger` is the best name for this parameter.